make/testing_flags.mk

```diff
@@ -34,12 +34,12 @@ UNIT_TARGETED ?= no
 # If a specific package/test case was requested, run the unit test for the
 # targeted case. Otherwise, default to running all tests.
 ifeq ($(UNIT_TARGETED), yes)
-UNIT := $(GOTEST) -tags="$(TEST_TAGS)" $(TEST_FLAGS) $(UNITPKG)
-UNIT_RACE := $(GOTEST) -tags="$(TEST_TAGS)" $(TEST_FLAGS) -race $(UNITPKG)
+UNIT := $(GOTEST) -tags="$(TEST_TAGS) nolog" $(TEST_FLAGS) $(UNITPKG)
```

Comment:
Commit message is a bit confusing, does this apply to non-main package unit tests also?

Reply:
this will apply to all packages, only in unit testing. i've now made this configurable via the `log=xxx` argument, which defaults to `nolog`, but can be overriden with a non-empty value

Reply:
Ok, nice :) This commit can be removed then?